WordPress/Classes/ViewRelated/NUX/SigninWPComViewController.swift

```diff
@@ -0,0 +1,279 @@
+import UIKit
+import WordPressComAnalytics
+import WordPressShared
+
+/// Provides a form and functionality for signing a user in to WordPress.com
+///
+@objc class SigninWPComViewController : NUXAbstractViewController, SigninWPComSyncHandler, SigninKeyboardResponder
+{
+
+    @IBOutlet weak var usernameField: WPWalkthroughTextField!
+    @IBOutlet weak var passwordField: WPWalkthroughTextField!
+    @IBOutlet weak var submitButton: NUXSubmitButton!
+    @IBOutlet weak var statusLabel: UILabel!
+    @IBOutlet var bottomContentConstraint: NSLayoutConstraint!
+    @IBOutlet var verticalCenterConstraint: NSLayoutConstraint!
+    var onePasswordButton: UIButton!
+
+    var immediateSignin = false;
+
+    lazy var loginFacade: LoginFacade = {
+        let facade = LoginFacade()
+        facade.delegate = self
+        return facade
```

Comment:
Nice catch. I was able to reproduce this, thanks for the steps!  
I don't think it was the delegate.  The delegate has a weak reference so it shouldn't need to be set to nil. When testing it looked like the error vc was presented from the view controller that was being dismissed, which would leave things in a weird state for sure, but deinit wouldn't have been called at this point.  I've changed things so the error vc is instead presented from the nav controller. It would be better if LoginFacade had a halt/cancel/stop method we could call but that's out of scope here.